frontend/dashboard/src/mocks/mockKeycloak.ts

```diff
@@ -0,0 +1,24 @@
+const createMockJwt = (payload: object) => {
+  const header = btoa(JSON.stringify({ alg: "HS256", typ: "JWT" }));
+  const body = btoa(JSON.stringify(payload));
+  return `${header}.${body}.signature`;
+};
+
+const mockToken = createMockJwt({
+  sub: "mock-user",
+  preferred_username: "mockuser",
+  email: "mock@diamond.ac.uk",
+  exp: Math.floor(Date.now() / 1000) + 3600,
```

Comment:
To make tests deterministic, maybe set to Number.MIN_SAFE_INTEGER and Number.MAX_SAFE_INTEGER